examples/pihole/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.5"
 
 services:
   pihole:
-    image: pihole/pihole:latest
+    image: pihole/pihole:beta-v5.0
```

Comment:
Version 5 is already out of beta phase. You can keep the latest tag for it! 🙂

Reply:
done!